assets/js/util/i18n.js

```diff
@@ -180,10 +203,9 @@ export const numFmt = ( number, options = {} ) => {
 
 	if ( 'metric' === style ) {
 		return readableLargeNumber( number );
-	}
-
-	if ( 'duration' === style ) {
-		return prepareSecondsForDisplay( number );
+	} else if ( 'duration' === style ) {
+		formatOptions = omit( options, 'style' );
```

Comment:
Let's remove the `omit` here – `durationFormat` should simply ignore/override it if passed.